extensions/skyportal/skyportal/app_server_fritz.py

```diff
@@ -0,0 +1,40 @@
+from skyportal.app_server import make_app
+
+from skyportal.handlers.api.alert import (
```

Comment:
Let's see, so these `skyportal`s are being imported from two different places, right? `make_app` is coming from the submodule, but `handlers.api.alert` is coming from the extension. What assumptions have to be made about the user path for this to work?

Reply:
No, the `fritz` script first copies over the extensions to the submodule directories - see [here](https://github.com/fritz-marshal/fritz/blob/76b8429a277a555e05a3b9917caa822191ab8254/fritz#L135) so the imports here are from the same place.